egs/chime4/asr1_multich/conf/preprocess.json

```diff
@@ -0,0 +1 @@
+{"process": [{"type": "stft", "win_length": 400, "n_fft": 512, "n_shift": 160, "window": "han"}]}
```

Comment:
Can nfft also be "400" as time resolution is only "400", we don't get any additional freq. resolution by having nfft "512". Maybe "400" will use normal DFT while "512" might use FFT algorithm. Does "512" make it faster to compute features online?

Reply:
This is the default parameter in Kaldi with fs == 16kHz.
FFT algo can be efficient calculation with the power of 2 number of fft, so Kaldi uses the nearest number to "400", and each window are zero padded so that it will be 512. This is a well known technique.

Reply:
Yes FFT works only with a power of 2 but DFT can be taken for any points. I don't think Kaldi uses mag spec without Mel filter banks as a feature so maybe it doesn't make any difference for them but for us we are increasing the input dimensions in the enhancement network from 400 to 512 without adding any frequency resolution. While running I don't see any time efficiency difference while using 400 vs 512, so maybe it's ok to use 400.

Reply:
In general, there are big difference especially if nfft is more large, but I agree that there are very  small difference if nfft==512. Actually, 512 might be slower due to zero padding about STFT case.

However, for me, either two case, 512 or 400, is o.k. In my experiments, both values achieved comparable results.
What do you care about this?

Reply:
Yes even I got comparable results with both. I would prefer 400 because it has lesser parameters in the enhancement network (lesser input and output dimensions for the masking network).

Reply:
Alright. I changed 400 to the default.
I preferred the compatibility to Kaldi, but your pointing is more important.